app/build.gradle.kts

```diff
@@ -43,7 +43,23 @@ android {
         generateLocaleConfig = true
     }
 
+    signingConfigs {
+        getByName("debug") {
+            keyAlias = "androiddebugkey"
+            keyPassword = "android"
+            storeFile = file("../keystores/debug.keystore")
```

Comment:
I generated a new keystore for this. We had talked a bit about using the same debug keystore as the main bitwarden app, but ultimately I'd prefer to keep them separate and protect access to the BridgeService with `knownSigner`. 

What I like about that is that the setup for `debug` and `release` will be the same: the `BridgeService` is protected behind `knownSigner`.

Reply:
Yeah I think that's a good plan.